docs/docs/overview/performance.md

```diff
@@ -67,23 +68,22 @@ Deployments: Constellation has the lowest latency for all cases except for scali
 #### Network
 
 There are two main indicators for network performance: intra-node and inter-node transmission speed.
-K-Bench provides benchmark tests for both, configured as `dp_netperf_internode`, `dp_network_internode`, and `dp_network_intranode`.
+K-Bench provides benchmark tests for both, configured as `dp_network_internode` and `dp_network_intranode`. The tests use [`iperf`](https://iperf.fr/) to measure the bandwidth available.
 
 ##### Inter-node
 
-K-Bench has two benchmarks to evaluate the network performance between different nodes.
+Inter-node communication is the network transmission between different Kubernetes nodes.
 
-The first test (`dp_netperf_internode`) uses [`netperf`](https://hewlettpackard.github.io/netperf/) to measure the throughput. Constellation has a slightly lower network throughput than AKS and GKE.
-This can largely be attributed to its [network encryption](../architecture/networking.md).
+The first test (`dp_network_internode`) measures the throughput between nodes. Constellation on Azure and on GCP has a steady inter-node throughput of around 870 Mbps. While that's faster than the average throughput of AKS at 577 Mbps, GKE provides faster networking.
```

Comment:
Please mention the throughput of GKE as well, feels weird that it's the only one missing the concrete value..

Reply:
Same in the archived ofc